Source/Model/SPTooltip.m

```diff
@@ -85,43 +85,73 @@ - (void)initMeWithOptions:(NSDictionary *)displayOptions;
 
 @implementation SPTooltip
 
+
++ (instancetype)sharedInstance {
+	static id sharedInstance = nil;
```

Comment:
why not `SPTooltip` but `id`?

Reply:
habit .. makes it easier to subclass, but in the this case we won't be subclassing. I'll make it explicit.